### DISCUSSION TITLE
GitHub student package benefits

### DISCUSSION THREAD

**Discussion by amr-ahmed23** [community/community#146110] [GitHub Education]
### Hi everyone,

i got accepted in the student package but i didn't get my benefits yet , i know i would get it within 3 days but I'm really in a hurry and i need it today because i have exams and i need to use intellij and I'm in it by GitHub student account  , and thank you ❤️

**Comment by quoc10973** [comment]
Keep waiting, you'll get it

**Comment by davevad93** [comment]
Hi @dank-m ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @dank-m was approved and your benefits are now available.`

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
